db/migrate/20220713141250_create_like.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
```

Comment:
- [ ] in the likes table the should be a reference to the author and the posts.

Reply:
Hello @mikethreels , Thanks for the professional review I follow all requirements and I changed it as mentioned . Thanks you can now check